llvm/docs/ReleaseNotes.md

```diff
@@ -575,6 +575,10 @@ Changes to LLDB
      24   int main() {
   Likely cause: f->b->d accessed 0x4
   ```
+* LLDB now supports hardware watchpoints for AArch64 Windows targets. Windows
+  does not provide API to query the number of supported hardware watchpoints.
+  Therefore current implementation allows only 1 watchpoint, as tested with
+  Windows 11 on the Microsoft SQ2 and Snapdragon Elite X platforms.
```

Comment:
What about this issue that's causing the test failures, is this something users are going to hit?

Or was that only about breakpoints, it's still in the PR description so I'm not sure.

Reply:
Only thing I can think of is perhaps `check-lldb` at `-j1` manages to start the new processes faster than running the test using lit alone. You could try running one of the tests using lit, in a loop.

But it's not like we don't have watchpoint glitches elsewhere, and if it's a kernel problem, this PR is certainly not going to fix that. So I think we say this is one more reason this feature is experimental.